.buildkite/pipelines/run_es_tests_aarch64.yml.sh

```diff
@@ -13,7 +13,7 @@ steps:
   - label: "Java :java: Integration Tests for aarch64 :hammer:"
     key: "java_integration_tests_aarch64"
     command:
-      - "sudo yum -y install java-17-amazon-corretto-devel"
+      - 'sudo mkdir -p /usr/lib/jvm/amazon-corretto-21 && wget -O - https://corretto.aws/downloads/latest/amazon-corretto-21-aarch64-linux-jdk.tar.gz | sudo tar xzf - -C /usr/lib/jvm/amazon-corretto-21 --strip=1'
```

Comment:
Can't you just do:
```
sudo yum install java-21-openjdk
```
See: https://docs.redhat.com/en/documentation/red_hat_build_of_openjdk/21/html-single/installing_and_using_red_hat_build_of_openjdk_21_on_rhel/index

Reply:
Thanks @jan-elastic ! I'll merge the current PR as it's working and take a look at your (much simpler!) suggestion.

Reply:
sounds good to me